lib/mjsonwp/routes.js

```diff
@@ -9,7 +9,7 @@ const METHOD_MAP = {
     GET: {command: 'getStatus'}
   },
   '/wd/hub/session': {
-    POST: {command: 'createSession', payloadParams: {required: ['desiredCapabilities'], optional: ['requiredCapabilities', 'capabilities']}}
+    POST: {command: 'createSession', payloadParams: {required: [['desiredCapabilities'], ['capabilities']], optional: ['requiredCapabilities']}}
```

Comment:
i don't see where in this PR you added support for treating the value of `required` as a disjunctive array?